packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol

```diff
@@ -294,6 +320,35 @@ contract StateCommitmentChain is IStateCommitmentChain, Lib_AddressResolver {
         emit StateBatchDeleted(_batchHeader.batchIndex, _batchHeader.batchRoot);
     }
 
+    /**
+     * Checks that a batch header matches the stored hash for the given index.
+     * @param _batch Submit batch data.
+     * @return _shouldStartAtElement or not the header matches the stored one.
+     */
+    function _distributeTssReward(bytes32[] calldata _batch, uint256 _shouldStartAtElement) internal {
+        // get address of tss group member
+        address[] memory tssMembers = ITSSGroupContract(resolve("TSSGroupContract")).GetTssMembers();
+        require(tssMembers.length > 0, "get tss members in error");
+
+        // construct calldata for claimReward call
+        bytes memory message = abi.encodeWithSelector(
+            ITssRewardContract.claimReward.selector,
+            _shouldStartAtElement,
```

Comment:
the `_shouldStartAtElement` may be not the same as the the `_blockStartHeight` in L2 TssRewardContract

Reply:
if _shouldStartAtElement == L2 height, it will work.